pkg/bundle/bundle.go

```diff
@@ -35,51 +34,28 @@ import (
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
+	"github.com/cert-manager/trust-manager/cmd/trust-manager/app/options"
 	trustapi "github.com/cert-manager/trust-manager/pkg/apis/trust/v1alpha1"
 	"github.com/cert-manager/trust-manager/pkg/bundle/internal/ssa_client"
 	"github.com/cert-manager/trust-manager/pkg/bundle/internal/target"
-	"github.com/cert-manager/trust-manager/pkg/fspkg"
 )
 
-// Options hold options for the Bundle controller.
-type Options struct {
-	// Log is the Bundle controller logger.
-	Log logr.Logger
```

Comment:
suggestion: I don't think it's required for this PR, but I'd love to see the logger pulled out of `Options` and passed around in the context. But it's not that important.

Reply:
Funny that you comment on this - as I have always been annoyed with why `Context` is not used to store/fetch a logger in this project. Since you seem open to this change, I will open a PR to fix this - before I make this PR ready for another round of review. 😸